recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "ruamel.yaml" %}
-{% set version = "0.16.12" %}
+{% set version = "0.17.21" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ruamel.yaml-{{ version }}.tar.gz
-  sha256: 076cc0bc34f1966d920a49f18b52b6ad559fbe656a0748e3535cf7b3f29ebf9e
+  sha256: 8b7ce697a2f212752a35c1ac414471dc16c424c9573be4926b56ff3f5d23b7af
 
 build:
-  number: 3
+  number: 0
```

Comment:
Skip `py<35`, see https://sourceforge.net/p/ruamel-yaml/code/ci/0.17.21/tree/__init__.py#l36

Reply:
It's better to mention the upstream constraints, not ours.

Reply:
ok thanks